listenstore/listenstore/listenstore.py

```diff
@@ -258,6 +218,98 @@ def fetch_listens_for_range(self, uid, date_range, from_id, to_id, order, limit=
             else:
                 return
 
+class PostgresListenStore(ListenStore):
+    def __init__(self, conf):
+        ListenStore.__init__(self, conf)
+        self.postgre_connection = None
+        self.postgre_cursor = None
+        
+        self.postgres_conn = psycopg2.connect("dbname='' user='listenbrainz'")   
```

Comment:
as I said in irc, make sure that you switch to sqlalchemy. Configuration should be in `config.py`, not in the file